plugins/search/api-report.md

```diff
@@ -210,15 +211,19 @@ export const SearchType: ({
   defaultValue,
 }: SearchTypeProps) => JSX.Element;
 
+// Warning: (ae-forgotten-export) The symbol "SidebarSearchProps" needs to be exported by the entry point index.d.ts
 // Warning: (ae-missing-release-tag) "SidebarSearch" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const SidebarSearch: () => JSX.Element;
+export const SidebarSearch: (props: SidebarSearchProps) => JSX.Element;
 
+// Warning: (ae-forgotten-export) The symbol "SidebarSearchModalProps" needs to be exported by the entry point index.d.ts
```

Comment:
didn't know why it didn't removed even after I re-run `yarn build:api-reports`

Reply:
Check out `src/index.ts` in the package. Normally, it would have an `export * from './components';` in it, but instead of an asterisk catch-all, it duplicates all of the actual names. So you will either have to change that to a star export (and double check in the generated api-report that nothing NEW inadvertently appears that's unrelated to your PR), or add your new names to its explicit list of names.